specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/AuthConfigs.json

```diff
@@ -174,6 +174,9 @@
         "x-ms-examples": {
           "Create or Update Container App AuthConfig": {
```

Comment:
```suggestion
          "Create or Update Container App AuthConfig with sas url": {
```

Reply:
this is the default auth config example without tokenStore

Reply:
Could you add an explicit mention like 
```suggestion
          "Create or Update Container App AuthConfig without token store": {
```